posthog/hogql_queries/insights/trends/aggregation_operations.py

```diff
@@ -181,6 +181,26 @@ def get_outer_aggregation(self, field: ast.Expr, is_histogram_breakdown: bool =
             return ast.Call(name="avg", args=[field])
         return ast.Call(name="sum", args=[field])
 
+    def get_array_fold_merge_operation(self) -> str:
+        """
+        Returns the ClickHouse expression for combining two values in an arrayFold operation
+        when rolling up "Other" breakdown values.
+
+        Used in arrayFold like: arrayMap(i -> {operation}, range(...))
+        where acc[i] and x[i] are the two values to combine.
+        """
+        if self.series.math == "max":
+            return "greatest(acc[i], x[i])"
+        elif self.series.math == "min":
+            return "least(acc[i], x[i])"
+        elif self.series.math == "avg":
+            # For avg, we'd need to track counts separately, so we fall back to sum for now
+            # This is a known limitation - averaging averages is not statistically sound anyway
```

Comment:
I think we should throw a validation error for these unsupported cases. In addition to avg, also p99 etc don't work.

See the validation error here https://github.com/PostHog/posthog/pull/43388/changes.

Averages don't work for the "other" breakdown, but work for the binning due to same sized buckets.